pkg/nsx/services/vpc/builder.go

```diff
@@ -89,3 +93,21 @@ func buildNSXVPC(obj *v1alpha1.NetworkInfo, nsObj *v1.Namespace, nc common.VPCNe
 
 	return vpc, nil
 }
+
+func buildNSXLBS(obj *v1alpha1.NetworkInfo, nsObj *v1.Namespace, cluster, lbsSize, vpcPath string, relaxScaleValidation *bool) (*model.LBService, error) {
+	lbs := &model.LBService{}
+	lbsName := util.GenerateDisplayName("", "vpc", nsObj.GetName(), "", cluster)
+	// LBS id should equal VPC id
```

Comment:
Why LBS id should equal VPC id?

Reply:
This is just a definition for the auto-created NSXLB.
Changed to: `Use VPC id for auto-created LBS id`